app/uk/gov/hmrc/customsservicestatus/controllers/CustomsServiceStatusController.scala

```diff
@@ -37,10 +36,9 @@ class CustomsServiceStatusController @Inject()(customsServiceStatusService: Cust
       customsServiceStatusService
         .updateServiceStatus(serviceName, state.state)
         .fold(
-          error =>
-            error match {
-              case LoadServicesConfigError   => ServiceUnavailable
-              case ServiceNotConfiguredError => NotFound(Json.toJson(State(s"Service with name $serviceName not configured")))
+          {
+            case LoadServicesConfigError   => ServiceUnavailable
+            case ServiceNotConfiguredError => NotFound(Json.toJson(State(s"Service with name $serviceName not configured")))
```

Comment:
Might be worth undoing this change to avoid a conflict on merge as the same change has been made on RORO-4581 (which will probably be merged first).